examples/aiservices/handel.yml

```diff
@@ -19,5 +19,6 @@ environments:
       - rekognition
       - polly
       - comprehend
+      - translate #translate is having an error and not deploying with the rest of the apigateway
```

Comment:
Did you mean to leave this comment on this line? Or was that something that was broken and now working?